tasks/add_new_prebid_partner.py

```diff
@@ -301,10 +339,42 @@ def main():
     len(placements)
   )
 
+  # In the case where no inventory is being used, make sure at least one 
+  # creative is created
+  if not num_creatives > 0:
+    num_creatives = 1
+
   bidder_code = getattr(settings, 'PREBID_BIDDER_CODE', None)
   if bidder_code is None:
     raise MissingSettingException('PREBID_BIDDER_CODE')
 
+  bidder_params = getattr(settings, 'PREBID_BIDDER_PARAMS', None)
+  hb_pb_key = 'hb_pb'
+  additional_keys = ''
+
+  if bidder_params is True:
+    hb_pb_key += '_' + bidder_code
```

Comment:
As noted elsewhere, please move this logic into a standalone module and reuse it elsewhere.